tools/regression_test.sh

```diff
@@ -127,7 +127,7 @@ function main {
       tmp=$DB_PATH
       DB_PATH=$ORIGIN_PATH
       test_remote "test -d $DB_PATH"
-      if [[ $? -ne 0 ]] || [[ $(run_remote 'echo $(( $(date +"%s") - $(stat -c "%Y" '"$DB_PATH"') ))') -gt "604800" ]]; then
+      if [[ $? -ne 0 ]] || [[ $(run_remote \''echo $(( $(date +"%s") - $(stat -c "%Y" '"$DB_PATH"') ))'\') -gt "604800" ]]; then
```

Comment:
IMO, I don't think you should be using single quotes here. Better to embed the whole thing in double quotes: 
http://unix.stackexchange.com/questions/118433/quoting-within-command-substitution-in-bash .